geonode/api/api.py

```diff
@@ -252,17 +252,17 @@ def dehydrate_email(self, bundle):
     def dehydrate_layers_count(self, bundle):
         obj_with_perms = get_objects_for_user(bundle.request.user,
                                               'base.view_resourcebase').instance_of(Layer)
-        return bundle.obj.resourcebase_set.filter(id__in=obj_with_perms.values('id')).distinct().count()
+        return bundle.obj.resourcebase_set.filter(id__in=obj_with_perms.values_list('id')).distinct().count()
```

Comment:
django is fine in using values('id') to filter with __in, adding the values_list is a small overhead